students/HarveyHerela/lesson04/mailroom2.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+
+donors = [{"firstname": "Chase", "lastname": "Zuma", "donations": [25]},
```

Comment:
If you were to use a `dict` for your outer data structure here with a unique key (full name?) for each 'record', you could simplify some of your look up code and input prompting loops below